### DISCUSSION TITLE
微调文字识别模型之后，原来能识别的文字也识别不准了

### DISCUSSION THREAD

**Discussion by xiaotangwan** [PaddlePaddle/PaddleOCR#13216] [Q&A]
我对字符集增加了一些不能识别的字符，如$这种，但是训练好的模型对原来官方模型能识别的字符也不能识别了。
我看了一下，只要我微调的时候没有包括的字符，都不能识别准确？我想问一下大家有遇到过这个问题吗？我觉得可能是微调的图片数量太少导致的，但我不知道多少能够可以。

**Comment by GreatV** [comment]
可以参考：https://github.com/PaddlePaddle/PaddleOCR/blob/main/doc/doc_ch/FAQ.md#25-%E9%A2%84%E8%AE%AD%E7%BB%83%E6%A8%A1%E5%9E%8B%E4%B8%8E%E5%BE%AE%E8%B0%83
